tyson/internal/tsembed/eval.go

```diff
@@ -0,0 +1,16 @@
+package tsembed
+
+import "github.com/dop251/goja"
+
+func evalJS(code string) (goja.Value, error) {
+	vm := goja.New()
```

Comment:
What's `vm`? `value map`? `virtual machine`?

Reply:
"virtual machine" (as in the virtual machine that interprets the javascript)